static/sass/styles.scss

```diff
@@ -821,6 +822,16 @@ td.p-accordion {
   z-index: 10;
 }
 
+#contact-success {
+  display: none;
+  position: absolute;
+}
+
+#contact-success:target {
+  display: block;
+  z-index: 10;
+}
```

Comment:
Are these used anywhere? I can't find any code that uses `#contact-success`

Reply:
Also, in the case we are actually using these styles, can you move them to `_pattern_modals.scss`, would make more sense if they are used across all form modals to be there?

Reply:
This is a confirmation banner for landscape form. The banners themselves are a part of base_index, so all banner styles are saved in styles, so I added it there too